meson.build

```diff
@@ -435,6 +435,7 @@ if get_option('enable-test')
   testenv.set('NNSTREAMER_FILTERS', path_nns_plugin_filters)
   testenv.set('NNSTREAMER_DECODERS', path_nns_plugin_decoders)
   testenv.set('NNSTREAMER_CONVERTERS', path_nns_plugin_converters)
+  testenv.set('NNSTREAMER_SOURCE_ROOT_PATH', meson.source_root())
```

Comment:
how about adding other test-env also? (C-Api, Movidius, and others in meson)

Reply:
@jaeyun-jung Thanks. Fixed.